.env.local

```
@@ -101,3 +101,5 @@ WALLETV3_API_UAT_BASEURL='http://169.254.191.117:3000'
 CIE_LOGIN_WITH_DEV_SERVER_ENABLED=NO
 # Redesign of the PN message details screen
 NEW_PN_MESSAGE_DETAILS_ENABLED=YES
+# Enable CDU flow new screen
+CDU_NEW_FLOW=YES
```

```suggestion
CDU_NEW_FLOW=NO
```

I'd leave the old flow for now. Even if we are in dev mode.